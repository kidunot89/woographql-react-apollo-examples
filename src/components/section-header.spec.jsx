import React from 'react';
import '@testing-library/react/dont-cleanup-after-each';
import { render, cleanup } from '@testing-library/react';

import SectionHeader from './section-header';

describe('SectionHeader', () => {
  afterAll(() => cleanup());

  const { baseElement } = render(
    <SectionHeader>Test</SectionHeader>,
  );

  it('should render successfully', () => {
    expect(baseElement).toMatchSnapshot();
  });
});