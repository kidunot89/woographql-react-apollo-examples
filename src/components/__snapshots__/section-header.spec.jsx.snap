// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHeader should render successfully 1`] = `
<body>
  <div>
    <div
      class="block font-semibold mb-2 text-lg"
    >
      Test
    </div>
  </div>
</body>
`;
